package.json

```diff
@@ -33,7 +33,8 @@
 	"files": [
 		"cli.js",
 		"index.js",
-		"lib"
+		"lib",
+		"vendor/src-pngquant/pngquant-2.10.1-src.tar.gz"
```

Comment:
Maybe just add `vendor/src-pngquant`? One less place to change when updating to a newer version.